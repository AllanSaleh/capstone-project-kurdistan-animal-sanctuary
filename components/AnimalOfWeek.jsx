import React from "react";

import Image from "next/image";

import catPhoto from "../public/Group 790.png";

export default function AnimalOfWeek() {
  return (
    <div
      style={{ backgroundImage: "url(https://i.ibb.co/zGrMtLL/paw.png)" }}
      className=" bg-no-repeat  bgSize	"
    >
      <div className="block p-7">
        <h1 className="text-center text-primary text-3xl font-Quicksand font-bold">
          Animal of the week
        </h1>
        <div className="flex">
          <div className="w-9/12">
            <Image src={catPhoto} />
          </div>
          <p className="text-center font-Quicksand">
            lorem ipsum dolor sit amet lorem, consectetur adipiscing elits
            lorem. lorem ipsum dolor sit amet lorem ipsum dolor sit amet lorem,
            consectetur adipiscing elits lorem. lorem ipsum dolor sit amet lorem
            ipsum dolor sit amet lorem, consectetur adipiscing elits lorem.
            lorem ipsum dolor sit amet lorem ipsum dolor sit amet lorem,
            consectetur adipiscing elits lorem. lorem ipsum dolor sit amet lorem
            ipsum dolor sit amet lorem, consectetur adipiscing elits lorem.
            lorem ipsum dolor sit amet lorem ipsum dolor sit amet lorem,
            consectetur adipiscing elits lorem. lorem ipsum dolor sit amet lorem
            ipsum dolor sit amet lorem, consectetur adipiscing elits lorem.
            lorem ipsum dolor sit amet lorem ipsum dolor sit amet lorem,
            consectetur adipiscing elits lorem. lorem ipsum dolor sit amet lorem
            ipsum dolor sit amet lorem, consectetur adipiscing elits lorem.
            lorem ipsum dolor sit amet lorem ipsum dolor sit amet lorem,
            consectetur adipiscing elits lorem. lorem ipsum dolor sit amet lorem
            ipsum dolor sit amet lorem, consectetur adipiscing elits lorem.
            lorem ipsum dolor sit amet lorem ipsum dolor sit amet lorem,
            consectetur adipiscing elits lorem. lorem ipsum dolor sit amet
          </p>
        </div>
      </div>
    </div>
  );
}
