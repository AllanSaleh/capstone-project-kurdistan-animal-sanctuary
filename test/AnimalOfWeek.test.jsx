import React from "react";
import renderer from "react-test-renderer";
import { test, expect } from "@jest/globals";
import AnimalOfWeek from "../components/AnimalOfWeek";

test("proparly renders the AnimalOfWeek component", () => {
  const tree = renderer.create(<AnimalOfWeek />).toJSON();
  expect(tree).toMatchSnapshot();
});
