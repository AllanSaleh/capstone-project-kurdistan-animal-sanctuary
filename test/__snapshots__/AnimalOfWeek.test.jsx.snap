// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`proparly renders the AnimalOfWeek component 1`] = `
<div
  className=" bg-no-repeat  bg-contain 	"
  style={
    Object {
      "backgroundImage": "url(https://i.ibb.co/zGrMtLL/paw.png)",
    }
  }
>
  <div
    className="block p-7 px-4 md:px-24"
  >
    <h1
      className="pb-8 text-center text-primary text-3xl font-Quicksand font-bold"
    >
      Animal of the week
    </h1>
    <div
      className="block md:flex"
    >
      <div
        className="w-36 md:w-44 h-36 md:h-44 rounded-full mx-auto   "
      >
        <img
          alt="animal profile "
          className="  w-32 md:w-36 h-32 md:h-36 border-2 border-highlight p-2 rounded-full cursor-pointer"
          onClick={[Function]}
          onMouseEnter={[Function]}
          src="./Group 790.png"
        />
      </div>
      <p
        className=" mx-auto md:mx-0  font-Quicksand text-justify w-10/12"
      >
        everyone meet Yoda! our perfect pet of the week, yoda was found when he was a little shy of a week two weeks old he fall down a roof and was abandoned by his mom but dont let that sad backstory distract you from the fact that yoda grew to be a gentle giant he is about a meter tall and likes to play with EVERYTHING just grab something tall and wave it and thats it Yoda will run toward you for play, now lets hear some facts about yoda, numbrt onr yoda doesnt know to hiss the farthest he will go when he is mad is to open his mouth  and do a sigh, number two everyday you have to hold and pet him while he pets you back by licking your arms. And thats it for out animal of the week hope you guys enjoyed getting to know yoda
      </p>
    </div>
  </div>
</div>
`;
